src/Lumi/Components2/Clip.purs

```diff
@@ -72,12 +73,14 @@ clip =
                   , css:
                     theme
                       # toCSS (Styles.Box.box <<< Styles.Box._justify Center)
-                      <> props.css
+                      <> toCSS (S.style_ (S.css { flex: S.str "0 0 calc(100% - 64px)", minWidth: S.str "0" }))
                   , ref
                   , children: props.content
                   }
               , box
                   $ _justify Center
+                  $ _align End
+                  $ S.style_ (S.css { flex: S.str "0 0 64px", minWidth: S.px 64 })
```

Comment:
The `minWidth` here might be redundant, not a big deal though.

It might be nice to put this `64` in a variable so the relationship to the styles above is clear.

Reply:
It's to override the default `min-width: min-content` set on our Boxes. Good call, I can break out the `64` into a var.

Reply:
Ahh, right, thanks!